pkg/asset/cluster/tfvars.go

```diff
@@ -83,6 +83,9 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&installconfig.ClusterID{},
 		&installconfig.InstallConfig{},
+		// Include PlatformProvisionCheck here but not in Generate
+		// as it performs validation but is not used.
+		&installconfig.PlatformProvisionCheck{},
```

Comment:
so we have added this to tfvars and cluster asset, do both of these need this or only one?

Reply:
I removed it from tfvars.. We cache the vsphere image in the tfvars step, but on closer inspection that doesn't need any of this validation.